templates/Courier2.html

```diff
@@ -122,57 +122,74 @@ <h3 style="margin-left:20px;margin-top:40px;">Available routes</h3>
 		// Initialize and add the map
 		function initMap() {
 			var directionsService = new google.maps.DirectionsService();
-			var directionsRenderer = new google.maps.DirectionsRenderer();
+			// var directionsRenderer = new google.maps.DirectionsRenderer();
 		// The location of Uluru
 
 		// The map, centered at Uluru
 			var map = new google.maps.Map(document.getElementById('map'), {zoom: 2, center: start});
 
-			directionsRenderer.setMap(map);
-
+			// directionsRenderer.setMap(map);
+			var starts = []
+			var finishes = []
+			console.log(locations[0][0].toString())
```

Comment:
This is how I know that both the start points are the same @wnuelle . We might be pulling incorrectly from the db